src/app/pages/Customize.svelte

```diff
@@ -125,17 +146,46 @@
   }
   main {
     max-width: var(--laptop);
-    min-height: 100vh;
   }
 
   .customize {
     display: flex;
     flex-direction: row;
     width: 100%;
-    height: 100%;
+    min-height: 100vh;
     font-family: Cabin, sans-serif;
   }
 
+  .customize__sidebar {
+    background-color: var(--gray-dark);
+  }
+
+  .customize__sidebar--project {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 2.5rem;
+    height: 2.5rem;
```

Comment:
I was wondering why a single letter needed a flex layout for itself, and then I realized that it's to center it. This works, but a flex layout for a single layout is a bit extra. Here's an alternative that uses vanilla CSS 2 properties (1990s CSS), but `rem` units, because that makes it more robost!

```suggestion
    width: 2.5rem;
    height: 2.5rem;
    
    line-height: 2.5rem;
    text-align: center;
    vertical-algim: middle;    
```